source/behavior/entities/format/components/projectile.json

```diff
@@ -285,9 +294,17 @@
           "properties": {
             "catch_fire": {
               "title": "Catch Fire",
-              "type": "boolean",
-              "default": false,
-              "description": "Determines if the struck object is set on fire."
+              "description": "Determines if the struck object is set on fire.",
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "fire_affected_by_griefing": {
+                  "title": "Fire Affected By Griefing",
+                  "description": "If the griefting gamerule can affect if fire is spawned",
+                  "type": "boolean",
+                  "default": true
+                }
+              }
             },
```

Comment:
Do you have an example of this working in game 🤔 
projectile is very badly documented and often very wrongly configured in vanilla files.

Reply:
I'll test it out in game and see if it is affected by griefing, otherwise I'll undo it.

Reply:
Tried it out and it doesn't work